### ISSUE TITLE
[4.0.0] Indent: comment lint error

### ISSUE THREAD

**Issue by chosh31** [eslint/eslint#8735]
**Tell us about your environment**

* **ESLint Version:** 4.0.0
* **Node Version:** 8.1.1
* **npm Version:** 5.0.3

**What parser (default, Babel-ESLint, etc.) are you using?**
 default

**What did you do? Please include the actual source code causing the issue.**

error code
```js
testClass = function() {
/*! comment 1 */
	this.foo = 'bar';
/*! comment 2 */
	this.foo = 'baz';
/*! comment 3 */
};
```
vs. success code.
```js
testClass = function() {
/*! comment 1 */
	this.foo = 'bar';
	/*! comment 2 */
	this.foo = 'baz';
/*! comment 3 */
};
```

**What did you expect to happen?**

**What actually happened? Please include the actual, raw output from ESLint.**
error code
```js
 { ruleId: 'indent',
    severity: 2,
    message: 'Expected indentation of 1 tab but found 0.',
    line: 6,
    column: 1,
    nodeType: 'Block',
    source: '/*! comment 2 */',
    endLine: 6,
    endColumn: 1,
    fix: { range: [Array], text: '\t' } }
```

**Comment by not-an-aardvark** [comment]
Thanks for the report. This is working as intended -- see https://github.com/eslint/eslint/issues/8704.

**Comment by kaicataldo** [comment]
Closing this issue as it looks like the question has been answered. Please feel free to visit us in the [ESLint Gitter](https://gitter.im/eslint/eslint) if you have any other issues!
